src/proc_macro_input.rs

```diff
@@ -27,13 +27,13 @@ impl Input for TokenStream {
         // for that is still "semver-exempt" in `proc-macro2`.
         let last = range
             .nonempty()
-            .map(|r| r.last().no_proof())
-            .unwrap_or(range.past_the_end());
-        Self::source_info_point(input, range.first())..Self::source_info_point(input, last)
+            .map(|r| /*r.last().no_proof()*/ unimplemented!())
```

Comment:
I need to add a back-up-one-item API to `indexing-str`, we currently only have back-up-n-units.